projects/questionset-editor-library/src/lib/components/header/header.component.ts

```diff
@@ -13,6 +13,8 @@ import { NgForm } from '@angular/forms';
 })
 export class HeaderComponent implements OnDestroy, OnInit {
   @Input() pageId: any;
+  @Input() rejectComment: string;
+  @Input() questionSetId: string;
```

Comment:
Is the `questionSetId` getting used anywhere?

Reply:
this property removed.